dmotion:
  ankle_roll:
    data: [0, 1.0070618237284903, 1.7977866755644532, 2.4134855299334106, 3.2137366304848785, 5.5, 0, -0.0089244102847882978, -4.1428166905261605, -4.1283363008952936, -0.031782537855132631, 0, 0, 0, 0, 0, 0, 0]